api/main_endpoints/routes/User.js

```diff
@@ -509,4 +509,41 @@ router.post('/usersSubscribedAndVerified', function(req, res) {
     });
 });
 
+// For Club Revenue page 
+router.post('/countMembers', async (req, res) => {
```

Comment:
wanna try adding a unit test for this? see https://github.com/SCE-Development/Clark/blob/1ab954db473c9de8f7ab2cef2f8622a0df837635/test/api/User.js#L135